index.ts

```diff
@@ -139,16 +140,15 @@ async function start() {
       throw new Error(`Journey ${answers.Journey} does not have a callable function attached to it`)
     }
 
-    try { await selectedFn.fn() }
-    catch (e) {
-      console.error(`An error occured on the selected journey:`)
+    try { await selectedFn.fn() } catch (e) {
+      console.error('An error occured on the selected journey:')
       console.error(e)
       break
     }
 
     saveSession(session)
   }
-  console.log("Bye!")
+  console.log('Bye!')
 }
 
-start()
\ No newline at end of file
+void start()
```

Comment:
Why the void? I assume it's eslint doing this.

Reply:
Yeah, promises should be awaited unless explicitly not (and if so, stick a void before it)